website/content/about/working-groups/deaf-and-hard-of-hearing.md

```diff
@@ -0,0 +1,119 @@
+---
+title: "Deaf & Hard of Hearing Working Group"
+linkTitle: "Deaf & Hard of Hearing"
+url: /about/deaf-and-hard-of-hearing
+---
+
+{{% blocks/lead color="primary" align="left" %}}
+
+Want to participate in empowering deaf or hard of hearing individuals? We are here to help!
+
+Read our [Charter](https://github.com/cncf/tag-contributor-strategy/tree/main/deaf-and-hard-of-hearing)
```

Comment:
```suggestion
To learn more about our group, read our [Charter](https://github.com/cncf/tag-contributor-strategy/tree/main/deaf-and-hard-of-hearing)
```

Reply:
I will keep this one as is. It is a subtitle directly below the header and all the other WG pages have the same format: https://deploy-preview-547--cncf-contribute.netlify.app/about/deaf-and-hard-of-hearing/

Reply:
cc @CathPag

Reply:
ok